cylc/flow/workflow_files.py

```diff
@@ -1527,13 +1528,10 @@ def install_workflow(
             Another workflow already has this name (unless --redirect).
             Trying to install a workflow that is nested inside of another.
     """
-    if not source:
-        source = Path.cwd()
-    elif Path(source).name == WorkflowFiles.FLOW_FILE:
-        source = Path(source).parent
-    source = Path(expand_path(source))
+    if source.name == WorkflowFiles.FLOW_FILE:
+        source = source.parent
```

Comment:
In light of #4816 this should be treated as an error?

Reply:
It might make sense to do so in #4816. Whichever PR goes in first will cause a merge conflict for the second (so it will be up to the author of the second to fix).

Reply:
Addressed by last commit in #4816